src/hooks/useTracking.js

```diff
@@ -103,6 +103,7 @@ export const useTracking = (
           session_id: getSessionId(),
           pageview_token: pageviewToken,
           client_type: 'kaios-app',
+          app_version: '1.0.0',
```

Comment:
We should probably read this from package.json

Reply:
Great! This is added here https://github.com/wikimedia/wikipedia-kaios/pull/131/commits/6be5f9a7dbcd634d3dd815123698da2d671a682f